.github/workflows/on_opened_pr.yml

```diff
@@ -12,13 +12,16 @@ jobs:
     uses: ./.github/workflows/reusable_export_pr_details.yml
     with:
       record_pr_workflow_id: ${{ github.event.workflow_run.id }}
+      workflow_origin: ${{ github.event.repository.full_name }}
     secrets:
       token: ${{ secrets.GITHUB_TOKEN }}
   check_related_issue:
     needs: get_pr_details
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
+      - name: "Debug workflow_run event"
+        run: echo ${{ toJSON(github.event) }}
```

Comment:
Should this be removed?

Reply:
YES! thank you

Reply:
it's coming from my fork, I'll rebase and drop it.